dockers/docker-ptf/Dockerfile.j2

```diff
@@ -217,6 +238,9 @@ RUN pip3 install setuptools \
     && pip3 install enum34 \
     && pip3 install grpcio \
     && pip3 install grpcio-tools \
+{% if PTF_ENV_PY_VER == "py3" %}
+    && pip3 install setuptools==51.0.0 \
+{% endif %}
```

Comment:
This pattern has been repeated multiple times. Is there a better way to install setuptools only once in the very end?

Reply:
I have tried to resolve this by using a j2 macro instead.